src/sqlfluff/dialects/dialect_ansi.py

```diff
@@ -759,13 +759,17 @@ class FunctionSegment(BaseSegment):
     type = "function"
     match_grammar = Sequence(
         Sequence(
-            Sequence(
```

Comment:
Rather than allowing 0 or 1 names in front of the function name, allow any number. (BigQuery only uses 2, so we could limit it to 2 if that's preferable.)

Reply:
I think we should probably have `allow_gaps=False` no? Unless there can be spaces between the elements. Otherwise I wonder whether this could behave strangely.

Reply:
I wasn't familiar with that property, but yes, that makes sense -- I assume it disallows white space? I'll add it.